crates/stackable-operator/src/commons/authentication/oidc.rs

```diff
@@ -132,10 +138,38 @@ impl AuthenticationProvider {
             })?;
         }
 
-        url.set_path(&self.root_path);
         Ok(url)
     }
 
+    /// Returns the OIDC endpoint [`Url`] without a trailing slash.
+    ///
+    /// To get the well-known URL, please use [`Self::well_known_url`].
```

Comment:
suggestion: Slight rewording.

```suggestion
    /// To retrieve the well-known OIDC configuration url, please use [`Self::well_known_config_url`].
```